src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -827,9 +854,17 @@ vector<Visitor::ExpressionPtr> Visitor::visitTypesafe(
   return visitVector(ctx->expression());
 }
 
+// ____________________________________________________________________________________
+void Visitor::visit(Parser::ExpressionListContext*) {
+  // ExpressionList can only come from RelationExpression or BuiltInCall both of
+  // which don't support the cases leading to ExpressionList.
```

Comment:
Use a consistent comment with the above `AD_FAIL` rules (I have written something there).